### PR TITLE
C109 pojo

### PR DESCRIPTION
**Pull request by HadarRose**
- create Tweet and User POJOs 
- changed TwitterRequestResource to expect POJOs instead of objects defined by twitter4j 
- changed TwitterResourceService to handle conversion from Status to Tweet 
- updated tests accordingly
- changed packaging structure

### CODE REVIEW

File: src/main/java/bootcamp/model/User.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
+package bootcamp.model;
+
+public class User {
```
Comment (amanduggal) [inline src/main/java/bootcamp/model/User.java:3]: look up what `@JsonIgnoreProperties` is. Very useful annotation that we use a lot when dealing with POJOs.

File: src/main/java/bootcamp/model/User.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
+package bootcamp.model;
```
Comment (amanduggal) [inline src/main/java/bootcamp/model/User.java:1]: there are some standard methods that are usually added (overridden) for all POJOs. See if you find what they are.

Comment (kev-chan) [reply src/main/java/bootcamp/model/User.java:1]: *upvote*

File: src/test/java/bootcamp/services/twitter4j/TwitterResourceServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.*;
+
+public class TwitterResourceServiceTest {
+    private Twitter mockedTwitter;
+
+    /* note to self about how I did this: extended a class that already implements most of the methods that are needed
+     * for ResponseList<T>*/
+    public class ResponseListStatus<T> extends ArrayList<T> implements ResponseList<T> {
```
Comment (kev-chan) [inline src/test/java/bootcamp/services/twitter4j/TwitterResourceServiceTest.java]: would not recommend nesting this class

**Review by kev-chan — APPROVED** [review]

**Pull request merged**